tasks/install-private-dependencies

```diff
@@ -0,0 +1,276 @@
+  uiDir
+],{
+  timeout: timeout
+});
+ifOKPrintIfErrPrintAndExit(out);
+process.stdout.write('done!\n');
+
+// check out specific commit
+process.stdout.write('Checking out commit ' + uiCommit + ' for ui-asq...');
+out = spawnSync('git', [
+  'checkout',
+  uiCommit
+],{
+  cwd: uiDir,
+  timeout: timeout
+});
+ifOKPrintIfErrPrintAndExit(out);
+process.stdout.write('done!\n');
+
+// install npm deps
+process.stdout.write('Installing npm dependencies for ui-asq...');
+out = spawnSync('npm', [
+  'install',
+  '--unsafe-perm'
+], {
+  cwd: uiDir,
+  timeout: timeout
+});
+ifOKPrintIfErrPrintAndExit(out);
+process.stdout.write('done!\n');
+
+// avoid using root .bowerrc by creating a local one
+// with the default settings
+process.stdout.write('Creating local .bowerrc...');
+let bowerrcContents = `{
+  "directory": "bower_components"
+}`;
+fs.writeFileSync(`${uiDir}/.bowerrc`, bowerrcContents);
+process.stdout.write('done!\n');
+
+// install bower deps
+process.stdout.write('Installing bower dependencies for ui-asq...');
+out = spawnSync('bower',[
+  'install',
+  '--allow-root'
+  ], {
+  cwd: uiDir,
+  timeout: timeout
+});
+ifOKPrintIfErrPrintAndExit(out);
+process.stdout.write('done!\n');
+
+// create .env for configuration
+process.stdout.write('Creating local .env..');
+const envContents = `ASQ=${config.rootUrl}`;
+fs.writeFileSync(`${uiDir}/.env`, envContents);
+process.stdout.write('done!\n');
+
+// build
+process.stdout.write('Building ui-asq...');
+out = spawnSync('npm',[
+  'run',
+  'build'
+  ], {
+  cwd: uiDir,
+  timeout: timeout
+});
+ifOKPrintIfErrPrintAndExit(out);
+process.stdout.write('done!\n');
+
+/* ------------ ui ------------ */
+
+/* ------------ cockpit ------------ */
+
+process.stdout.write('Cleaning cockpit-asq dir...');
+fs.removeSync(cockpitDir);
+process.stdout.write('done!\n');
+
+// download
+process.stdout.write('Downloading ' + cockpitBranch + ' branch for cockpit-asq...');
+out = spawnSync('git', [
+  'clone',
+  '-b',
+  cockpitBranch,
+  'https://' + process.env.CI_USER_TOKEN + ':x-oauth-basic@github.com/ASQ-USI/cockpit-asq.git',
+  cockpitDir
+],{
+  timeout: timeout
+});
+ifOKPrintIfErrPrintAndExit(out);
+process.stdout.write('done!\n');
+
+// check out specific commit
+process.stdout.write('Checking out commit ' + cockpitCommit + ' for cockpit-asq...');
+out = spawnSync('git', [
+  'checkout',
+  cockpitCommit
+],{
+  cwd: cockpitDir,
+  timeout: timeout
+});
+ifOKPrintIfErrPrintAndExit(out);
+process.stdout.write('done!\n');
+
+// install npm deps
+process.stdout.write('Installing npm dependencies for cockpit-asq...');
+out = spawnSync('npm', [
+  'install',
+  '--unsafe-perm'
+], {
+  cwd: cockpitDir,
+  timeout: timeout
+});
+ifOKPrintIfErrPrintAndExit(out);
+process.stdout.write('done!\n');
+
+// avoid using root .bowerrc by creating a local one
+// with the default settings
+process.stdout.write('Creating local .bowerrc...');
+bowerrcContents = `{
+  "directory": "bower_components"
```

Comment:
I copied it from your scripts, can you please fix this. I would like to avoid doing a mess.